app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
```

Comment:
do it in one line

Reply:
counter = sum(1 for char in phrase)
print(counter)

Reply:
do it, use count()